turbopack/crates/turbopack-ecmascript/src/tree_shake/mod.rs

```diff
@@ -233,7 +234,7 @@ impl Analyzer<'_> {
                         let state = self.vars.entry(id.clone()).or_default();
 
                         self.g.add_weak_deps(item_id, state.last_writes.iter());
-                        self.g.add_weak_deps(item_id, state.last_reads.iter());
+                        self.g.add_strong_deps(item_id, state.last_reads.iter());
```

Comment:
Without this change, some test cases fail.

Reply:
I'll try reverting it

Reply:
I got 

```
   [cause]: TypeError: Cannot read properties of undefined (reading 'call')
      at o.charToGlyphIndex (/private/var/folders/w8/mm77k2r1245cgm11qzzyzdsm0000gn/T/next-install-eaba06cef16ab9ed5682402a7bcb47c6cdb87ed4f6850343589478fe77658397/.next/server/edge/chunks/0d0f5_next_dist_compiled_@vercel_og_index_edge_5dcd1e.js:1:48909)
      at /private/var/folders/w8/mm77k2r1245cgm11qzzyzdsm0000gn/T/next-install-eaba06cef16ab9ed5682402a7bcb47c6cdb87ed4f6850343589478fe77658397/.next/server/edge/chunks/0d0f5_next_dist_compiled_@vercel_og_index_edge_5dcd1e.js:1:50033
      at Array.find (<anonymous>)
      at c (/private/var/folders/w8/mm77k2r1245cgm11qzzyzdsm0000gn/T/next-install-eaba06cef16ab9ed5682402a7bcb47c6cdb87ed4f6850343589478fe77658397/.next/server/edge/chunks/0d0f5_next_dist_compiled_@vercel_og_index_edge_5dcd1e.js:1:50017)
      at m (/private/var/folders/w8/mm77k2r1245cgm11qzzyzdsm0000gn/T/next-install-eaba06cef16ab9ed5682402a7bcb47c6cdb87ed4f6850343589478fe77658397/.next/server/edge/chunks/0d0f5_next_dist_compiled_@vercel_og_index_edge_5dcd1e.js:1:50360)
      at Object.has (/private/var/folders/w8/mm77k2r1245cgm11qzzyzdsm0000gn/T/next-install-eaba06cef16ab9ed5682402a7bcb47c6cdb87ed4f6850343589478fe77658397/.next/server/edge/chunks/0d0f5_next_dist_compiled_@vercel_og_index_edge_5dcd1e.js:2:18)
      at /private/var/folders/w8/mm77k2r1245cgm11qzzyzdsm0000gn/T/next-install-eaba06cef16ab9ed5682402a7bcb47c6cdb87ed4f6850343589478fe77658397/.next/server/edge/chunks/0d0f5_next_dist_compiled_@vercel_og_index_edge_f9228e.js:3:13196
      at Array.filter (<anonymous>)
      at tv (/private/var/folders/w8/mm77k2r1245cgm11qzzyzdsm0000gn/T/next-install-eaba06cef16ab9ed5682402a7bcb47c6cdb87ed4f6850343589478fe77658397/.next/server/edge/chunks/0d0f5_next_dist_compiled_@vercel_og_index_edge_f9228e.js:3:13175)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
}```

because `charToGlyphIndex` is a method, but it's added to the type using `xxx.prototype.charToGlyphIndex = function() {}`. That was the reasoning behind this change, but I forgot the exact reasoning so I tried reverting.